assignment.md

```diff
@@ -39,48 +39,40 @@ OVERALL QUESTIONS
 
-### Retrieving code updates (`fetch`)
-The `fetch` command first checks your local branch to determine whether an associated tracking branch exists remotely. If the tracking branch exists, `fetch` retrieves the changes and history from the tracking branch and pulls those changes into the <!-- tracking? remote? --> branch.
+### Retrieving updates from a branch (`git fetch`)
+The `git fetch` command retrieves the changes and history from a tracking branch to a locally-cached version of your remote repository. The `git fetch` command gives you an opportunity to review the changes that others have made to the tracking branch before merging them with the assets in your local branch. You can use `git fetch` to resolve conflicts before they are merged to your local branch.
 
-The `fetch` command does not update your local branch.
+The `git fetch` command does not update your local branch.
 
-<!-- EAJ QUESTIONS:
-    - The text that was originally here was confusing and a jumble of terminology. I am primarily trying to get straight what is updated and where. I understand that it downloads a branch from a remote repository (either the origin or an upstream branch), including the history of each asset. What I don't understand is when we say "retrieve" or "download," where is this downloaded to? It says my local branch isn't updated, but how can I review it locally if my local branch isn't updated? Is it updated on a remote repository?
-    - Does the user need to run fetch before they pull or push? How often?
--->
+To learn more about `git fetch`, review [Git's `git fetch` documentation](https://git-scm.com/docs/git-fetch).
+
+### Retrieving changes and merging them to your local repository (`git pull`)
+The `git pull` command fetches changes from a remote repository, then applies those changes to your local repository so that you work from the latest copy. The `git pull` command serves as the basis for which code comparisons are made before you push or merge local changes to a branch or repository.
+
+You must run `git pull` often to ensure that your local assets match the remote repository.
 
-### Retrieving changes and merging them to your local repository (`pull`)
-The `pull` command fetches changes from the tracking branch or repository, then applies those changes to your local repository so that you have the latest copy.
+To learn more about `git pull`, review [GitHub's Git Pull guide](https://github.com/git-guides/git-pull).
 
-### Sending your changes to the remote repository (`push`)
-The `push` command checks for a tracking branch or remote repository, then updates the remote branch with your changes.
+### Sending your changes to the remote repository (`git push`)
+The `git push` command updates the remote branch with your local changes. To ensure that you can successfully push updates to the remote branch, pull the latest version of your objects from the repository or branch.
```

Comment:
Adding this information about `pull` is super useful to avoid problems before they arise.